packages/eslint-config-cozy-app/basics.js

```diff
@@ -19,6 +19,7 @@ module.exports = {
         semi: false
       }
     ],
-    'no-param-reassign': 'warn'
+    'no-param-reassign': 'warn',
+    'spaced-comment': ['error','always']
```

Comment:
It's very easy to do the change if necessary.

Something like `codemod --extensions js,jsx "//([a-zA-Z0-9]O" "// \1"` should be enough.

So I think it's ok to make it an error.

Reply:
lint --fix should fix issues too ;)